help/en/releasenotes/current-draft-note.shtml

```diff
@@ -515,7 +515,9 @@
     <h3>Throttle</h3>
         <a id="throttle" name="throttle"></a>
         <ul>
-            <li></li>
+            <li>When "Large speed slider" mode is enabled (in throttle preferences), directions, stop, estop buttons will automaticaly depending on the control panel inner window size, allows for high visibility from far away (for people using a wireless device combined JMRI native throttles for instance)</li>
```

Comment:
I have trouble parsing this sentence.  There seems to be a missing verb between 'automatically' (spelling) and 'depending'.

Reply:
fixed